media-build-logic/src/main/kotlin/publication-convention.gradle.kts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+import java.util.Base64
+
+plugins {
+    id("javadoc-stub-convention")
+    id("org.gradle.maven-publish")
+    id("signing")
+}
+
+publishing {
+    repositories.maven("https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/") {
+        name = "OSSRH"
+
+        credentials {
+            username = System.getenv("OSSRH_USER")
+            password = System.getenv("OSSRH_KEY")
+        }
+    }
+
+    publications.withType<MavenPublication> {
+        // Provide artifacts information requited by Maven Central
+        pom {
+            name.set("MOKO media")
+            description.set("Media selection & presenting for mobile (android & ios) Kotlin Multiplatform development")
+            url.set("https://github.com/icerockdev/moko-media")
+            licenses {
+                license {
+                    name.set("Apache-2.0")
+                    distribution.set("repo")
+                    url.set("https://github.com/icerockdev/moko-media/blob/master/LICENSE.md")
+                }
+            }
+
+            developers {
+                developer {
+                    id.set("Alex009")
+                    name.set("Aleksey Mikhailov")
+                    email.set("aleksey.mikhailov@icerockdev.com")
+                }
```

Comment:
not all developers. please check original configuration and copy-paste `pom` block